File: BeecrowdProblem/Salary.dart
import 'dart:io';

void main() {

  int number = int.parse(stdin.readLineSync()!);
  int hours = int.parse(stdin.readLineSync()!);
  double salaryPerHour = double.parse(stdin.readLineSync()!);

  double salary = hours * salaryPerHour;

  print("NUMBER = $number");
  print("SALARY = U\$ ${salary.toStringAsFixed(2)}");
}

File: BeecrowdProblem/Salary.kt
fun main(args: Array<String>) {

    val number = readLine()!!.toInt()
    val hours = readLine()!!.toInt()
    val salaryPerHour = readLine()!!.toDouble()

    val salary = hours * salaryPerHour

    println("NUMBER = $number")
    println("SALARY = U$ %.2f".format(salary))
}